server/model/tests.go

```diff
@@ -180,13 +180,20 @@ const (
 	RunStateCreated             RunState = "CREATED"
 	RunStateExecuting           RunState = "EXECUTING"
 	RunStateAwaitingTrace       RunState = "AWAITING_TRACE"
-	RunStateFailed              RunState = "FAILED"
```

Comment:
Today we save this value on our database, right? How will we keep the retro compatibility with older Tracetest versions?
Should we map the old `FAILED` string as always `ASSERTION_FAILED`?

Reply:
Or even writing a migration to fix it on the database?